packages/nx-plugin/src/generators/app/lib/create-project.ts

```diff
@@ -0,0 +1,150 @@
+import { NormalizedOptions } from '../generator';
+import type { ProjectConfiguration, Tree } from '@nrwl/devkit';
+import { addProjectConfiguration } from '@nrwl/devkit';
+
+export function createProject(
+  tree: Tree,
+  normalizedOptions: NormalizedOptions
+) {
+  const projectConfiguration: ProjectConfiguration = {
+    root: normalizedOptions.projectRoot,
+    projectType: 'application',
+    sourceRoot: `${normalizedOptions.projectRoot}/src`,
+    // todo: do we want a custom executor hiding most of this?
+    // targets: {
+    //   build: {
+    //     executor: '@analogjs/nx:build',
+    //   },
+    // },
+    targets: {
+      build: {
+        executor: '@nrwl/vite:build',
+        outputs: [
+          '{options.outputPath}',
+          `dist/apps/${normalizedOptions.projectName}/.nitro`,
+          `dist/apps/${normalizedOptions.projectName}/ssr`,
+          `dist/apps/${normalizedOptions.projectName}/analog`,
+        ],
+        options: {
+          configFile: 'vite.config.ts',
+          outputPath: `dist/apps/${normalizedOptions.projectName}/client`,
+        },
+        defaultConfiguration: 'production',
+        configurations: {
+          development: {
+            mode: 'development',
+          },
+          production: {
+            sourcemap: false,
+            mode: 'production',
+          },
+        },
+      },
+      serve: {
+        executor: '@nrwl/vite:dev-server',
+        defaultConfiguration: 'development',
+        options: {
+          buildTarget: `${normalizedOptions.projectName}:build`,
+          port: 4200,
+        },
+        configurations: {
+          development: {
+            buildTarget: `${normalizedOptions.projectName}:build:development`,
+            hmr: true,
+          },
+          production: {
+            buildTarget: `${normalizedOptions.projectName}:build:production`,
+          },
+        },
+      },
+      'build-ng': {
```

Comment:
`build-ng` was just for initial testing and should not be included

Reply:
Removed